### PR TITLE
issue-214: Update to Jakarta EE8 APIs

### LINKED ISSUE #214: Update to Jakarta EE8 APIs for MP 4.0 in Jun 2020

### PR DESCRIPTION
**Pull request by xstefank**
resolves #214

### CODE REVIEW

**Comment by kenfinnigan** [comment]
Should the dependencies be "provided"?

**Comment by xstefank** [comment]
@kenfinnigan wasn't before but it's a valid point. This should be required in all MicroProfile API artifacts.

File: pom.xml [hunk 1]
```diff
@@ -119,25 +119,28 @@
...
         <dependencies>
             <dependency>
-                <groupId>javax.enterprise</groupId>
-                <artifactId>cdi-api</artifactId>
+                <groupId>jakarta.enterprise</groupId>
+                <artifactId>jakarta.enterprise.cdi-api</artifactId>
                 <version>${cdi-api.version}</version>
+                <scope>provided</scope>
                 <exclusions>
                     <exclusion>
```
Comment (kenfinnigan) [inline pom.xml]: Based on other threads I've seen, does this also need an EJB exclude as it was added by mistake to Jakarta CDI?

Comment (xstefank) [reply pom.xml]: @kenfinnigan yes, thanks for pointing it out.

**Pull request merged**